rs/rosetta-api/icp_ledger/rosetta-integration-tests/tests/tests.rs

```diff
@@ -29,8 +29,8 @@ use tempfile::TempDir;
 use url::Url;
 
 pub const LEDGER_CANISTER_INDEX_IN_NNS_SUBNET: u64 = 2;
-const MAX_ATTEMPTS: u16 = 1000;
-const DURATION_BETWEEN_ATTEMPTS: Duration = Duration::from_millis(100);
+const MAX_ATTEMPTS: u16 = 10;
+const DURATION_BETWEEN_ATTEMPTS: Duration = Duration::from_millis(1000);
```

Comment:
This decreases the total time that the test can retry. Could this impact the flakiness of the test? Maybe something to at least keep an eye on?

Reply:
I decreased the number of attempts and increased the time it waits. If anything flakiness should decrease because the overall wait time decreased.